proseco/guide.py

```diff
@@ -97,6 +102,60 @@ def get_guide_catalog(obsid=0, **kwargs):
     return guides
 
 
+def get_guide_candidates_mask(
+    stars: "Table",
+    t_ccd: float = -10.0,
+    dyn_bgd: bool = True,
+) -> np.ndarray:
+    """Get filter on ``stars`` for acceptable guide candidates.
+
+    This does not include spatial filtering.
+
+    Parameters
+    ----------
+    stars : Table
+        Table of stars
+    t_ccd : float
+        ACA CCD temperature (degC)
+    dyn_bgd : bool
+        Dynamic background enabled flag
+
+    Returns
+    -------
+    ok : np.array of bool
+        Mask of acceptable stars
+    """
+    # Scale the reference ACA faint mag limit to the CCD temperature keeping
+    # expected signal-to-noise constant.
+    faint_mag_limit = snr_mag_for_t_ccd(
+        t_ccd, ref_mag=GUIDE.ref_faint_mag, ref_t_ccd=GUIDE.ref_faint_mag_t_ccd
+    )
+
+    # Without dynamic background, ensure faint limit is no larger (fainter)
+    # than ref_faint_mag (10.3 mag).
+    if not dyn_bgd:
+        faint_mag_limit = min(faint_mag_limit, GUIDE.ref_faint_mag)
+
+    # Allow for stars from proseco StarsTable.from_stars() with `mag` and `mag_err`,
+    # or native agasc.get_agasc_cone() stars table.
+    mag = stars["mag"] if "mag" in stars.colnames else stars["MAG_ACA"]
+    mag_err = (
+        stars["mag_err"] if "mag_err" in stars.colnames else stars["MAG_ACA_ERR"] / 100
+    )
+
+    ok = (
+        (stars["CLASS"] == 0)
+        & (mag > 5.2)
+        & (mag < faint_mag_limit)
+        & (mag_err < 1.0)
+        & (stars["ASPQ1"] < 20)  # Mag err < 1.0 mag
```

Comment:
I think the previous comments were wrong/shifted and that persists here.

Reply:
Fixed.